docs/guide/testing-types.md

```diff
@@ -60,7 +60,7 @@ This will pass, because it expects an error, but the word “answer” has a typ
 
 ```ts
 // @ts-expect-error answer is not a string
-assertType<string>(answr) //
+assertType<string>(answer) //
```

Comment:
Please, read the paragraph before this code example. This change is not needed.

Reply:
Oh, I'm sorry.